src/compiler/emitter.ts

```diff
@@ -120,7 +119,7 @@ var __param = this.__param || function(index, decorator) { return function (targ
 
             let generatedNameSet: Map<string> = {};
             let nodeToGeneratedName: string[] = [];
-            let blockScopedVariableToGeneratedName: string[];
+            let blockScopedNameToGeneratedName: string[];

```

Comment:
In the checker these are called blockScopedValues but here they are blockScopedNames. Is there a reason for the terminology difference?